flyway/migrations/V2__init.sql

```diff
@@ -23,12 +23,12 @@ BEGIN;
   INSERT INTO target_group
       (target_group_name, target_type, protocol, port, vpc, health_check_path)
   VALUES
-      ('${project_name}-target', 'ip', 'HTTP', ${port}, 'default', '/health');
+      ('${project_name}-target', 'ip', 'HTTP', ${port}, null, '/health');
```

Comment:
any reason we don't just omit the field here?

Reply:
I suppose we could? I just did the minimal code change :)